osdconfig/constants.go

```diff
@@ -0,0 +1,23 @@
+package osdconfig
+
+// kvdb keys
+const (
+	base_key    = "osdconfig"   // root of the tree in kvdb
```

Comment:
According to golang naming standards, variable names should be in camel case. No underscores or all caps.

Reply:
@disrani-px : glad you had a chance to review. will fix the items you noted shortly.

Reply:
done